app/code/Magento/SalesRule/Model/Coupon/UpdateCouponUsages.php

```diff
@@ -38,21 +40,29 @@ class UpdateCouponUsages
     private $couponUsage;
 
     /**
-     * @param RuleFactory $ruleFactory
+     * @var Customer
+     */
+    private $customerRuleUsage;
+
+    /**
+     * @param RuleFactory     $ruleFactory
      * @param CustomerFactory $ruleCustomerFactory
-     * @param Coupon $coupon
-     * @param Usage $couponUsage
+     * @param Coupon          $coupon
+     * @param Usage           $couponUsage
+     * @param Customer        $customerRuleUsage
      */
     public function __construct(
-        RuleFactory $ruleFactory,
+        RuleFactory     $ruleFactory,
         CustomerFactory $ruleCustomerFactory,
-        Coupon $coupon,
-        Usage $couponUsage
+        Coupon          $coupon,
```

Comment:
Please, remove extra spaces

Reply:
Hi @sidolov,
 I have removed the extra spaces. Kindly check and let me know.